src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -87,10 +87,11 @@ private void InitializeArgs(NativeCompileSettings config)
             }
 
             // AppDep Libs
+            var baseAppDeplibPath = Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04/x64");
```

Comment:
Since `ubuntu.14.04/x64` looks almost identical to the RID `ubuntu.14.04-x64`, can we use that instead? So the files would be located in `CPPSdk/ubuntu.14.04-x64/...`?

We generally use RIDs everywhere else (`corehost` build, `artifacts` directory, CoreFX packages, etc.), so it would be nice to try and keep everything else consistent.